docs/guides/context.md

```diff
@@ -67,17 +67,58 @@ type Post {
 
 In this case, `value.userId` is a way to get the `userId` information from the "parent" context of the `Post` type. Essentially, it's extracting a list or "array" of `userId` fields from multiple `Post` types. Think of `value` as a container that holds the results of a post query, with `userId` being the specific key you want to fetch from that container.
 
-### env
+### Environment Variables
 
-This represents global environment variables for the server. This is set once when the server starts.
+Environment variables allow you to configure different aspects of your Tailcall server without needing to hardcode values. Some common uses:
 
-```graphql showLineNumbers
-type Query {
-  users: [User]! @http(baseUrl: "{{env.API_ENDPOINT}}", path: "/users")
+Setting the base URL for API requests:
+```type Query {
+  users: [User]! @http(baseUrl: "{{env.API_ENDPOINT}}", path: "/users") 
 }
 ```
 
-In the above example, `env.API_ENDPOINT` refers to an environment variable called API_ENDPOINT, which should be defined in your server settings.
+Toggling features on/off:
+```type Query {
+  recommendations @include(if: "{{env.SHOW_RECOMMENDATIONS}}") {
+    products(first: 10)
+  }
+}
+```
+
+Setting API keys:
+```type Query {
+  weather(zip: String!): Weather
+    @http(headers: {apiKey: "{{env.WEATHER_API_KEY}}"}) 
+}
+```
+Environment variables are set when starting the Tailcall server and can be accessed in GraphQL using {{env.VAR_NAME}}.
+
+### Variables
+Variables allow passing dynamic values into queries. They can be used to make queries more reusable and configurable.
+
+For example:
+
+```query GetUser($id: ID!) {
+  user(id: $id) {
+    name
+  }
+}
+```
+
+When executing the query, the $id variable value is provided:
+```
+{
+  "id": "1234"
+}
+```
+Variables have the following properties:
+
+->Declared with $name syntax
+->Type needs to be provided
+->Passed as separate parameter to query
+->Can be used in place of literal values
+->Help avoid duplication in queries
+->Enable dynamic query construction.
```

Comment:
This looks incomplete.

Reply:
@tusharmath added some more features info. You can review . Thanks